src/context/index.tsx

```diff
@@ -18,25 +18,33 @@ type FoundryColorsType = Record<keyof typeof colorsEnum, string>;
 export type FoundryContextType = {
   globalStyles: string;
   colors: FoundryColorsType;
+  styleConstants: object;
 };
+
 const defaultContextValue = {
   globalStyles: defaultGlobalStyles,
   colors: colorsEnum,
+  styleConstants: {},
   // TODO Add Foundry's "theme" to items here and pull from the ContextProvider
 };
+
 export const FoundryContext = React.createContext<FoundryContextType>(defaultContextValue);
 
 export const FoundryProvider = ({
   value = defaultContextValue,
   children,
 }: {
-  value?: { globalStyles?: string; colors?: Partial<Record<keyof typeof colorsEnum, string>> };
+  value?: {
+    globalStyles?: string;
+    colors?: Partial<Record<keyof typeof colorsEnum, string>>;
+    styleConstants?: {};
+  };
   children: React.ReactNode;
 }) => {
-  const { globalStyles = defaultGlobalStyles, colors = colorsEnum } = value;
+  const { globalStyles = defaultGlobalStyles, colors = colorsEnum, styleConstants = {} } = value;
```

Comment:
I was kind of hoping for an implementation where the value object can have any attributes, kind of like this:
```suggestion
  const { globalStyles = defaultGlobalStyles, colors = colorsEnum, ...customConstants } = value;
```

It can be typed as well: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html

Reply:
Does that feel weird @jakeoien ?

Reply:
My initial thought is it makes more sense to have them behind a specific key in the context object. 

The existing Context behaves like a configuration object, where we're effectively changing "settings" by applying a global style and colors, and having a spread over "whatever else you put in here" seems like it doesn't match the other keys hierarchically. 

So I'd recommend to separate the name out, and it's possible that we can update components later with these sorts of style constants, and we have a key on the object already available to add those in there. Just seems cleaner to me without the spread in this case

Reply:
Exactly what I needed to hear, thanks!